app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
+  load_and_authorize_resource
+  before_action :set_user, only: %i[show follow unfollow]
+  #helper_method :current_user_owner?
+
+  def show
+    @user
+  end
+
+  def follow
+    if @user && !current_user.following_user?(@user)
+      current_user.follow(@user)
+      redirect_to "/users/show/#{@user.username}"
```

Comment:
@sansuaza absolute routes need to be changed by route helpers methods!

Reply:
solved!